src/zope/app/container/browser/tests/test_contents_functional.py

```diff
@@ -349,24 +345,33 @@ def test_copy_then_delete_with_unicode_name(self):
         response = self.publish('/@@contents.html', basic='mgr:mgrpw', form={
             'ids' : (u'voil\xe0',),
             'container_copy_button' : '' })
-        self.assertEqual(response.getStatus(), 302)
-        self.assertEqual(response.getHeader('Location'),
+        self.assertEqual(response.status_int, 302)
+        self.assertEqual(response.headers.get('Location'),
             'http://localhost/@@contents.html')
         response = self.publish('/@@contents.html', basic='mgr:mgrpw')
-        self.assertEqual(response.getStatus(), 200)
+        self.assertEqual(response.status_int, 200)
 
         # delete the object
         del root[u'voil\xe0']
         transaction.commit()
         response = self.publish('/@@contents.html', basic='mgr:mgrpw')
-        self.assertEqual(response.getStatus(), 200)
+        self.assertEqual(response.status_int, 200)
 
 
 def test_suite():
     suite = unittest.TestSuite()
     Test.layer = AppContainerLayer
     suite.addTest(unittest.makeSuite(Test))
-    index = FunctionalDocFileSuite("index.txt")
+    def _http(query_str, *args, **kwargs):
+        # Strip leading \n
+        wsgi_app = AppContainerLayer.make_wsgi_app()
+
+        query_str = query_str[1:]
```

Comment:
I *literally* cannot type that correctly. My fingers are hardwired to type 'sl' instead of 'ls' following a dot (but only a dot, for some reason, it's fine at the shell prompt) and I *always* have to back-up and start over. So I tend not to use it, which means I tend to forget about it, which means my fingers never get any better...